model/session.go

```diff
@@ -140,7 +141,35 @@ func (me *Session) GetTeamByTeamId(teamId string) *TeamMember {
 }
 
 func (me *Session) IsMobileApp() bool {
-	return len(me.DeviceId) > 0
+	return len(me.DeviceId) > 0 || me.IsMobile()
+}
+
+func (me *Session) IsMobile() bool {
+	val, ok := me.Props["isMobile"]
+	if !ok {
+		return false
+	}
+	isMobile, err := strconv.ParseBool(val)
+	if err != nil {
```

Comment:
If there's an error here it means there's some invalid value in the `Props` for the session. It's unlikely, but I think it's worth logging.

Reply:
Done!